packages/components/src/components/FileUploader/FileUploader.tsx

```diff
@@ -46,6 +46,9 @@ export interface FileUploaderProps
 
   /** Disables the file uploader */
   disabled?: boolean;
+
+  /** Handles clicking on file name anchor/link */
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void;
```

Comment:
Do you think it is a good name for this property? I mean, onClick is pretty generic, and this component has a lot of clickable elements. I wonder if it could create misunderstandings.

Reply:
I updated for `onLabelClick`, the reason is because the File title may also be a regular `span` element and that way it's more predictable about what this prop does.